src/epubParsers.js

```diff
@@ -52,35 +52,50 @@ exports.getBuffer = (stream) => {
   })
 }
 
-exports.epubStore = (fileName, itemID, updated, type, response) => {
+exports.epubStore = (partName, instanceID, updated, type, response, itemData, fileName) => {
   logger.info('Storing file in S3')
```

Comment:
you might want to add some identifier information to the log message here to distinguish which file is being stored.